modules/ROOT/pages/_partials/sync/server-data-access.adoc

```diff
@@ -64,12 +64,8 @@ const resolvers = //define the resolvers
 const apolloConfig = {
   typeDefs,
   resolvers,
-  context: async ({ req }) => {
-    // pass request + db ref into context for each resolver
-    return {
-      req: req,
-      db: db.client
-    }
+  context: {
+    db: db.client
```

Comment:
I think this may be just `db`as this comes from old format we had in server app:
https://github.com/aerogear/ionic-showcase/blob/master/server/src/index.js#L39

and above we have direct access to db.